config/config-package.py

```diff
@@ -78,7 +83,8 @@ def call(*args, capture_output=False):
          'setup.cfg', 'tox.ini', '.gitignore', '.travis.yml', 'MANIFEST.in',
          '.editorconfig')
     call('git', 'ci', '-m', f'Configuring for {config_type}')
```

Comment:
So, uh, not related to this PR/commit, but `git ci` is not recognized by git, unless you have an alias in your ~/.gitconfig.  It would be better to use `git commit`.

Reply:
Good catch!

Reply:
Thank you, fixed in 1f7a4a2